.github/workflows/update-libs.yaml

```diff
@@ -0,0 +1,46 @@
+name: Auto-update Charm Libraries
+on:
+  # Manual trigger
+  workflow_dispatch:
+  # Check regularly the upstream every four hours
+  schedule:
+    - cron: "0 0,4,8,12,16,20 * * *"
+
+jobs:
+  update-lib:
+    name: Check libraries
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744 # v3
+        with:
+          fetch-depth: 0
+
+      # FIXME: Enable this when we can configure CHARMHUB_TOKEN
+      # - name: Check libs
+      #   run: |
+      #     sudo snap install charmcraft --classic --channel latest/stable
+      #     charmcraft fetch-lib
+      #   env:
+      #     CHARMCRAFT_AUTH: "${{ secrets.CHARMHUB_TOKEN }}"
+
+      - uses: tibdex/github-app-token@b62528385c34dbc9f38e5f4225ac829252d1ea92 # v1
```

Comment:
Is this an action which can be trusted? We pass it our Github token so we better should. Is this used by other charms?

Reply:
This auto update workflow is something I still need to take a look at. The parca-k8s and zinc-k8s operators have this, although I don't think we need this right now as we do not have charm libraries as of such, so I feel I should get rid of this workflow altogether.